Makefile

```diff
@@ -1,3 +1,9 @@
+build:
+	go build .

```

Comment:
It builds only go binary but not assets. Web UI will not works fine

Reply:
Good catch - thanks @makhov. I'll need to change this to a `buffalo build`

Reply:
done in https://github.com/gomods/athens/pull/60/commits/74e74ef65fa374ddc82204ca016803772add0300